engine/src/main/scala/cromwell/engine/workflow/ValidateActor.scala

```diff
@@ -36,17 +34,18 @@ object ValidateActor {
 }
 
 // TODO: Declarations cannot be validated here currently because of it's dependency on EngineFunctions, and in turn on IOManager and WfContext
-class ValidateActor()
-  extends Actor with LazyLogging {
+class ValidateActor() extends Actor with LazyLogging {
 
-  import ValidateActor.ValidateWorkflow
+  import ValidateActor.{EmptyJson, ValidateWorkflow}
   import context.dispatcher
 
   override def receive = {
     case ValidateWorkflow(wdlSource, workflowInputs, workflowOptions) =>
       val requester = sender()
-        // `validateAll(..)` guarantees we will get a `ValidationResult`, and nothing else
-      validateAll(wdlSource, workflowInputs, workflowOptions) map { requester ! _ }
+      // `validateAll(..)` guarantees we will get a `ValidationResult`, and nothing else
```

Comment:
Currently a `Future[ValidationResult]`, but maybe it should be just a `ValidationResult` :smile: